resources/views/about/for-regulated-organizations.blade.php

```diff
@@ -11,18 +11,30 @@
     </x-slot>
 
     <div class="-mb-8 space-y-16 px-0">
-        <x-placeholder />
+        <div class="stack w-full" x-data="vimeoPlayer({
+            url: @if (locale() === 'en') 'https://vimeo.com/789854286'
+                @elseif (locale() === 'fr')
+                'https://vimeo.com/789770856'
+                @elseif (locale() === 'asl')
+                'https://vimeo.com/788820695/f357173576'
+                @elseif (locale() === 'lsq')
+                'https://vimeo.com/789829171' @endif,
+            byline: false,
+            pip: true,
+            portrait: false,
+            responsive: true,
+            speed: true,
+            title: false
+        })" @ended="player().setCurrentTime(0)">
+        </div>
 
         <x-section class="stack:lg" aria-labelledby="what">
             <div class="text-center">
                 <h2 id="what">{{ __('What you can do on this website') }}</h2>
                 <x-interpretation name="{{ __('What you can do on this website', [], 'en') }}" />
             </div>
 
-            <x-media-text>
-                <x-slot name="media">
-                    <x-placeholder width="915" height="515" />
-                </x-slot>
+            <div class="grid">
```

Comment:
From the [wireframes](https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=1125%3A3283&t=yhQZiynmAWlXonZN-0) this should be a 2x2 grid, but am getting 3 in the first row and 1 in the last.

<img width="1616" alt="Grid displayed with 3 in first row and 1 item in second" src="https://user-images.githubusercontent.com/556024/214046957-5641d232-0bbc-4630-bc7d-6edf754bef09.png">

Reply:
Same applies to the for-community-organizations page.

Reply:
Hey @jobara we talked about responsible 2 column layout before when I was working on the training module dashboard, and I couldn't figure out best way to do 2 column layout that is responsible in different viewport sizes. I know we could potentially use media queries, but is that something we want to customize? For the training module dashboard part, I just told Cheryl that I will go with our current grid layout, where it fits all possible content, 3 in this case.

Reply:
@cherylhjli can you confirm if it's okay to be 3 columns in this case for now. We could file a new issue to look more into this later. If not, we'll see if we can figure something out now.

Reply:
3 columns is fine @jobara !